documentation/status-logging.md

```diff
@@ -0,0 +1,54 @@
+# Status logging
+
+TK-status can periodically log status data to a file in JSON format.
+
+To take advantage of this you must do two things:
+
+1. Enable logging though you TK app's configuration. For example, to log every 2 seconds, your configuration might look like this:
+
+    ```
+    status: {
+        logging-frequency: 2000,
+        status-logging-enabled: true
+    }
```

Comment:
Yeah this is new. For sure, I just figured I'd use it to get started.

Interval, that's the word I was looking for.

I'd be fine either way with regards to the "enabled" setting